integration/__specs__/modules/components/cartPage/cartModal/cartNameInput/cartNameInput.ts

```diff
@@ -0,0 +1,11 @@
+import { Component } from '@Core/component';
+
+export class CartNameInput extends Component {
+    protected selectors = {
+        nameInput: './/input[@name="name"]',
+    };
+
+    public async getNameInput() {
+        await this.element.waitForXpath(this.selectors.nameInput);
+    }
+}
```

Comment:
We don't need this component, since we create them using 'Component' in cartModal.ts